platform/android/CHANGELOG.md

```diff
@@ -4,8 +4,12 @@ Mapbox welcomes participation and contributions from everyone.  If you'd like to
 
 ## master
 
+## 8.5.0-alpha.1 - October 3, 2019
+[Changes](https://github.com/mapbox/mapbox-gl-native/compare/android-v8.4.0...android-v8.5.0-alpha.1) since [Mapbox Maps SDK for Android v8.4.0](https://github.com/mapbox/mapbox-gl-native/releases/tag/android-v8.4.0):
 ### Bug fixes
  - Suppress network requests for expired tiles update, if these tiles are invisible. [#15741](https://github.com/mapbox/mapbox-gl-native/pull/15741)
+ - Fixed opacity interpolation for composition expressions [#15738](https://github.com/mapbox/mapbox-gl-native/pull/15738)
+ - Remove projected meters per pixel and pixel ratio relation [#15742](https://github.com/mapbox/mapbox-gl-native/pull/15742)
```

Comment:
Actually, there's a suggested changelog entry in the PR https://github.com/mapbox/mapbox-gl-native/pull/15742#issuecomment-537448116

Reply:
Changed.